shared/challenge_builder.py

```diff
@@ -90,7 +90,9 @@ def queryElementsRaw(self, overpass_query):
         return response.json()["elements"]
 
     def queryElementsAsGeoJSON(self, overpass_query, forceGeomType=None):
-        return [self.geoJSONGeometryFromOverpassElement(element, forceGeomType) for element in self.queryElementsRaw(overpass_query)]
+        rawElements = self.queryElementsRaw(overpass_query)
+        geometriesAndProperties = [(self.geoJSONGeometryFromOverpassElement(element, forceGeomType)) for element in rawElements]
+        return [geojson.Feature(geometry=g, properties=p) for g,p in geometriesAndProperties]
```

Comment:
`geoJSONGeometryFromOverpassElement` I don't get why a function named `geoJSONGeometryFromOverpassElement` should return a tuple containing the tags as well.

Read the tags inside `queryElementsAsGeoJSON` and combine these two for loops into one.

Reply:
Done like that